e2e/legacy/controller.yaml

```diff
@@ -0,0 +1,6 @@
+- op: replace
+  path: /spec/podInfoOnMount
+  value: false
+
+- op: remove
+  path: /spec/volumeLifecycleModes
```

Comment:
I cannot find any references of `spec.volumeLifecycleModes` in the current manifests.
Why this `op` is needed?

Reply:
I added this to```spec.volumeLifecycleModes``` in e2ebase. I haven't touched the ```deploy/manifests``` yet. Should the changes I made in e2ebase instead go in the ```deploy/manifests```? Perhaps along with an update to https://github.com/cybozu-go/topolvm/blob/master/deploy/README.md instructing users to remove this if they are deploying on k8s versions prior to 1.16?

Reply:
Ah, got it.  Thanks.